x-pack/legacy/plugins/canvas/canvas_plugin_src/uis/arguments/toggle.js

```diff
@@ -26,6 +26,8 @@ const ToggleArgInput = ({ onValueChange, argValue, argId, renderError }) => {
         checked={argValue}
         onChange={handleChange}
         className="canvasArg__switch"
+        aria-label={typeInstance.displayName}
+        showLabel={false}
```

Comment:
This `showLabel=false` is causing some warning in the console

`EuiSwitch 'label' must be a string when 'showLabel' is false. `